### PR TITLE
EL-865 -- Update to add subscribers only when we selected the 'subscribe to newsletter'

### PR DESCRIPTION
**Pull request by henrys-cm**

### CODE REVIEW

File: core/App.php [hunk 1]
```diff
@@ -274,13 +274,7 @@ public static function checkout_process($orderId){
             }
 
             if (array_key_exists('cmw_register_email', $_POST)){
-                if (!empty($email)){
```
Comment (henrys-cm) [inline core/App.php:277]: Essentially, previously there is this 2 ifs statement --> had combined them and only actioned if the end user select the option as true.

File: core/App.php [hunk 2]
```diff
@@ -329,6 +324,9 @@ public static function checkout_process($orderId){
                 Log::write($result);
 
             }
+            }
```
Comment (OliverToonCM) [inline core/App.php:327]: Nit: the indentation on these lines is not enough

**Review by OliverToonCM — APPROVED** [review]
LGTM! EL.

File: core/App.php (hunk 2 above)
Comment (henrys-cm) [reply core/App.php]: Thanks... updated =)

**Pull request merged**